Dockerfile

```diff
@@ -74,6 +75,23 @@ ARG NGX_BROTLI_COMMIT
 ARG HEADERS_MORE_VERSION
 ARG CONFIG
 
+ARG MAXMIND_VERSION=1.2.1
+
+#Install GeoIp2. Thanks to https://github.com/bubelov/nginx-alpine-geoip2/blob/master/Dockerfile
+RUN set -x \
+  && apk add --no-cache --virtual .build-deps \
+    alpine-sdk \
+    perl \
+  && git clone https://github.com/leev/ngx_http_geoip2_module /ngx_http_geoip2_module \
```

Comment:
Can we rather pin to a specific version of this module? I'd rather have repeatable builds here.

Reply:
should be resolved now.

Reply:
Thanks!